src/Api/OpenApi/OpenApiDecorator.php

```diff
@@ -44,6 +47,11 @@ public function normalize($object, $format = null, array $context = [])
         // "scopes" should be object, not array
         $docs['components']['securitySchemes']['oauth']['flows']['password']['scopes'] = new \stdClass();
 
+        // Set max identifier count from .env
+        $description = sprintf($docs['paths']['/api/v2/covers']['get']['parameters'][1]['description'], $this->maxIdentifierCount);
+        $docs['paths']['/api/v2/covers']['get']['parameters'][1]['description'] = $description;
+        $docs['paths']['/api/v2/covers']['get']['parameters'][1]['schema']['maxLength'] = 10;
```

Comment:
Should the 10 have been `$this->maxIdentifierCount`?

Reply:
Fixed